File: csharp/main/Implementation/LiteralUnresolved.cs
using System;

namespace Deltix.Luminary.Implementation
{
	internal class LiteralUnresolved : Literal, ILiteral<String>
	{
		public LiteralUnresolved(LiteralKind kind, String value)
			: base(kind)
		{
			Value = value;
		}

		public String Value { get; }

		public override String ToString()
		{
			return Value;
		}
	}
}

File: java/src/main/java/deltix/luminary/implementation/LiteralUnresolved.java
package deltix.luminary.implementation;

import deltix.luminary.Literal;
import deltix.luminary.LiteralKind;

public class LiteralUnresolved extends Literal {
    private final String value;

    protected LiteralUnresolved(LiteralKind kind, String value) {
        super(kind);
        this.value = value;
    }

    public String getValue() {
        return value;
    }

    @Override
    public String toString() {
        return value;
    }
}
